src/test/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProjectTest.java

```diff
@@ -194,7 +194,7 @@ public DescriptorImpl() {
         mp.getSourcesList().add(new BranchSource(new GitSCMSource(null, sampleRepo.toString(), "", "*", "", false), new DefaultBranchPropertyStrategy(new BranchProperty[0])));
         mp.getSourcesList().add(new BranchSource(new GitSCMSource(null, sampleRepo.toString(), "", "*", "", false), new DefaultBranchPropertyStrategy(new BranchProperty[0])));
         WorkflowJob p = scheduleAndFindBranchProject(mp, "master");
-        mp.getIndexing().writeWholeLogTo(System.out);
+        r.waitUntilNoActivity();
```

Comment:
nit: I would leave the old code in as well to help diagnose any future issues:

```suggestion
        mp.getIndexing().writeWholeLogTo(System.out);
        r.waitUntilNoActivity();
```

Reply:
It's actually redundant.  `scheduleAndFindBranchProject()` calls down to eventually call `showIndexing()`.